client/state/data-layer/wpcom/jetpack/settings/index.js

```diff
@@ -138,7 +138,7 @@ export const handleSaveFailure = (
 ) => {
 	dispatch( updateJetpackSettings( siteId, previousSettings ) );
 	dispatch(
-		errorNotice( translate( 'An unexpected error occurred. Please try again later.' ), {
+		errorNotice( translate( 'There was a problem saving your changes. Please try again.' ), {
```

Comment:
I think so, don't you? After all, we're showing these errors only upon attempting to save something on the server side...

Reply:
Well, it makes sense when saving the site title step form. But it would be a little odd on some steps, for example:

* The "Contact form" step - the user would think "hey, I am trying to insert a contact page, not to save anything?"
* The "WooCommerce" step - the user would think "hey, I'm trying to install WooCommerce and not to save anything?"

So, it might be a good idea to keep it more generic and universal.

Reply:
Okay, reverted the relevant commit.